src/common/react-novnc.js

```diff
@@ -56,9 +57,31 @@ export default class NoVnc extends Component {
         onClipboardChange(text)
       })
     })
+
+    rfb._onUpdateState = (rfb, state) => {
```

Comment:
Pass the callback as a constructor option `onUpdateState` ([source](https://github.com/kanaka/noVNC/wiki/ModulesAPI#rfb-module)).